client/components/Cart.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { connect } from "react-redux";
+import { deleteItem } from "../store/cartReducer";
+
+class Cart extends React.Component {
+  render() {
+    const { cart } = this.props;
+    const subTotal = 0;
+    return (
+      <div>
+        {cart.items ? (
+          <div>
+            {cart.items.map((item) => {
```

Comment:
with such a big block of code like this, it's probably better to separate this into another component so it wouldnt be too overwhelming